lib/core/text/flat.nit

```diff
@@ -85,6 +85,104 @@ redef class FlatText
 		return ns_i
 	end
 
+	# By escaping `self` to HTML, how many more bytes will be needed ?
+	fun chars_to_html_escape: Int do
```

Comment:
The class is private, so the method is not public either :)
On 20 Oct 2015 7:54 pm, "Alexis Laferrière" <notifications@github.com>
wrote:

> In lib/core/text/flat.nit
> <https://github.com/nitlang/nit/pull/1775#discussion_r42569819>:
>
> > @@ -85,6 +85,104 @@ redef class FlatText
> >  		return ns_i
> >  	end
> >
> > +	# By escaping `self` to HTML, how many more bytes will be needed ?
> > +	fun chars_to_html_escape: Int do
>
> Is this useful as a public method?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nitlang/nit/pull/1775/files#r42569819>.
>